flocker/node/agents/blockdevice.py

```diff
@@ -181,6 +184,12 @@ def __init__(self, blockdevice):
     u"The system device file for an attached block device."
 )
 
+PROFILE_NAME = Field(
+    u"profile_name",
+    identity,
```

Comment:
Probably better to use `Field.for_type` and limit this to unicode.  Anything else is a bug, right?